sunpy/__init__.py

```diff
@@ -48,4 +48,23 @@ class UnsupportedPythonError(Exception):
     # Load user configuration
     config = load_config()
 
+    import logging
+
+    # Use the root logger as a dummy log before initializing Astropy's logger
+    log = logging.getLogger()
+
+    from astropy.logger import _init_log
```

Comment:
Can you make all this setup a function in `sunpy.util.logger` or something rather than having it all in the `__init__.py`?

Reply:
I can try to move it though it won't be as clean because I need the config read in to setup the logger correctly. I guess I could define a function which takes the config object as input?

Reply:
Moved it!